drivers/SmartThings/jbl/src/init.lua

```diff
@@ -0,0 +1,221 @@
+--  Copyright 2022 SmartThings
```

Comment:
nit: date (I think? This PR isn't _that_ old is it?)

Reply:
@cchillerr

Reply:
changed to 2023

Reply:
was this change missed in the pr update?

Reply:
Now it's available in the PR, please check.